packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -290,6 +290,7 @@ class MessageList extends Widget<Properties> {
         icon: 'edit',
         text: editText,
         disabled: isEditActionDisabled(message),
+        // @ts-expect-error ts-error
```

Comment:
A CollectionItem can include an onClick property (see the implementation in collection.base), but that property isn’t public. I think we should use our internal type for contextMenuItem instead of the public ContextMenuItem here, and our internal type should be extended with onClick property, after this the error will disappear